backend/lcfs/web/api/organizations/views.py

```diff
@@ -172,8 +172,9 @@ async def get_externally_registered_organizations(
     """
     return await service.get_externally_registered_organizations(org_id=request.user.organization_id)
 
+
 @router.get("/balances/{organization_id}", response_model=OrganizationBalanceResponseSchema, status_code=status.HTTP_200_OK)
-@roles_required("Government")
+@roles_required("Government", 'Supplier')
```

Comment:
We have to be careful here because this added Supplier role would allow any supplier access to other organization balances they are not apart of. There should be an endpoint specifically for Supplier org balances that gets the organization id from the request.user.organization object.